q2_types_genomics/reference_db/tests/test_format.py

```diff
@@ -88,3 +89,99 @@ def test_eggnog_dir_fmt(self):
+
+    def test_ncbi_tax_nodes_dmp_ok(self):
+        fp = self.get_data_path("ncbi/nodes-ok.dmp")
+        format = NCBITaxonomyNodesFormat(fp, "r")
+        format.validate()
+
+    def test_ncbi_tax_nodes_dmp_too_few_cols(self):
+        fp = self.get_data_path("ncbi/nodes-wrong-cols.dmp")
+        format = NCBITaxonomyNodesFormat(fp, "r")
+        with self.assertRaisesRegex(
+                ValidationError, r"found 12 columns on line 2."
+        ):
+            format.validate()
+
+    def test_ncbi_tax_nodes_dmp_nonnumeric_id(self):
+        fp = self.get_data_path("ncbi/nodes-non-numeric.dmp")
+        format = NCBITaxonomyNodesFormat(fp, "r")
+        with self.assertRaisesRegex(ValidationError, r"value on line 3."):
+            format.validate()
+
+    def test_ncbi_tax_nodes_dmp_nonnumeric_other(self):
+        fp = self.get_data_path("ncbi/nodes-non-numeric-other.dmp")
+        format = NCBITaxonomyNodesFormat(fp, "r")
+        with self.assertRaisesRegex(ValidationError, r"line 2, column 6: x."):
+            format.validate()
+
+    def test_ncbi_taxonomy_dir_fmt(self):
+        dirpath = self.get_data_path("ncbi/db-valid")
+        format = NCBITaxonomyDirFmt(dirpath, mode="r")
+        format.validate()
+
+    def test_binary_file_fmt_positive(self):
+        dirpath = self.get_data_path("ncbi/db-valid/prot.accession2taxid.gz")
+        format = NCBITaxonomyBinaryFileFmt(dirpath, mode="r")
+        format.validate()
+
+    def test_binary_file_fmt_wrong_col(self):
+        dirpath = self.get_data_path("ncbi/wrong_col.gz")
+        format = NCBITaxonomyBinaryFileFmt(dirpath, mode="r")
+        with self.assertRaises(ValidationError):
+            format.validate()
+
+    def test_binary_file_fmt_extra_col(self):
+        dirpath = self.get_data_path("ncbi/too_many_cols.gz")
+        format = NCBITaxonomyBinaryFileFmt(dirpath, mode="r")
+        with self.assertRaises(ValidationError):
+            format.validate()
+
+    def test_binary_file_fmt_wrong_accession(self):
+        dirpath = self.get_data_path("ncbi/wrong_accession.gz")
+        format = NCBITaxonomyBinaryFileFmt(dirpath, mode="r")
+        with self.assertRaises(ValidationError):
+            format.validate()
+
+    def test_binary_file_fmt_wrong_accession_version(self):
+        dirpath = self.get_data_path("ncbi/wrong_accession_version.gz")
+        format = NCBITaxonomyBinaryFileFmt(dirpath, mode="r")
+        with self.assertRaises(ValidationError):
+            format.validate()
+
+    def test_binary_file_fmt_wrong_taxid(self):
+        dirpath = self.get_data_path("ncbi/wrong_taxid.gz")
+        format = NCBITaxonomyBinaryFileFmt(dirpath, mode="r")
+        with self.assertRaises(ValidationError):
+            format.validate()
+
+    def test_binary_file_fmt_wrong_gi(self):
+        dirpath = self.get_data_path("ncbi/wrong_gi.gz")
+        format = NCBITaxonomyBinaryFileFmt(dirpath, mode="r")
+        with self.assertRaises(ValidationError):
```

Comment:
For all the ValidationError assertions you should include a check for the message content - see how it's done in the tests above. Without it, it's very difficult to see what the test is actually testing as the test files are GZipped.